pkgs/development/interpreters/renpy/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, stdenv, fetchFromGitHub, python3Packages, pkg-config, SDL2
+, libpng, ffmpeg, freetype, glew, libGL, libGLU, fribidi, zlib
+, glib
+}:
+
+with python3Packages;
+
+stdenv.mkDerivation rec {
+  pname = "renpy";
+  version = "8.0.0";
+
+  meta = with lib; {
+    description = "Ren'Py Visual Novel Engine";
+    homepage = "https://renpy.org/";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ shadowrz ];
+  };
+
+  src = fetchFromGitHub {
+    owner = "renpy";
+    repo = "renpy";
+    rev = "8.0.0.22062402";
+    sha256 = "sha256-37Hbs0i5eXMjVaETX7ImJCak0y8XtEHUaRFceA9J39A=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    python cython wrapPython tkinter future six pefile requests
+    SDL2 libpng ffmpeg freetype glew libGLU libGL fribidi zlib pygame_sdl2 glib
+  ];
+
+  pythonPath = [ pygame_sdl2 tkinter future six pefile requests ];
+
+  RENPY_DEPS_INSTALL = lib.concatStringsSep "::" (map (path: path) [
+    SDL2 SDL2.dev libpng ffmpeg ffmpeg.out freetype glew.dev glew.out libGLU libGL fribidi zlib
+  ]);
+
+  patchPhase = ''
+    cp tutorial/game/tutorial_director.rpy{m,}
+  '';
+
+  buildPhase = ''
+    python module/setup.py build
+  '';
```

Comment:
This ignores $NIX_BUILD_CORES.

Reply:
Changed to use `setuptoolsBuildHook` which should respect `$NIX_BUILD_CORES`.